### PR TITLE
Add code of conduct and writing style guide

### PR DESCRIPTION
**Pull request by mounderfod**
This is an important PR, so it will not be merged without at least 5 approvals.

### CODE REVIEW

**Review by sschr15 — APPROVED** [review]
As much as British English spelling will be a challenge for me, it all is reasonable, though I think there should be a request on markdown line length perhaps

File: WRITING-STYLE-GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+## 4. Use correct spelling, punctuation and grammar
+
+- **All sentences should be syntactically and grammatically correct**
+
+| Avoid                           | Prefer                            |
+| ------------------------------- | --------------------------------- |
+| _You should of gone outside._   | **You should have gone outside.** |
+| _She's married with a dentist._ | **She's married to a dentist.**   |
+
+- **In terms of spelling, use British English spelling over American English**
```
Comment (Jamalam360) [inline WRITING-STYLE-GUIDE.md:88]: image

**Review by Jamalam360 — APPROVED** [review]
lgtm

**Comment by Jamalam360** [comment]
Mentioning formatters would be good though

**Comment by mounderfod** [comment]
> Mentioning formatters would be good though

It's possible this is out of scope for this PR, as I'd ideally like to have a preset config for common formatters to define e.g. line length (as @sschr15 also mentioned).

**Comment by mounderfod** [comment]
> Should the use of AI be mentioned?

I hadn't considered this, although it is a very good point. Would this possibly necessitate a separate document outlying the protocol for dealing with potentially AI-written articles that are submitted?

**Comment by Jamalam360** [comment]
This has reviews from everyone who has been active so far, so I think we're good to merge it.

**Pull request merged**